### ISSUE TITLE
"-" instead of "="

### ISSUE THREAD

**Issue by aplab** [andreberg/Meslo-Font#24]
Phpstorm Meslo LG L Size 11 
displayed "-" instead of "="

example http://clip2net.com/s/3fiMtOe

**Comment by udeleng** [comment]
Yes, this is happening in IntelliJ for the Meslo LG M font in bold.
